From 29b3eb171ebc079deefc877f98f32d5f7d479113 Mon Sep 17 00:00:00 2001
From: Josef Schlehofer <pepe.schlehofer@gmail.com>
Date: Sun, 18 Apr 2021 18:15:04 +0200
Subject: [PATCH 1/2] reforis-distutils: remove it

This package was created to create and compile host variant of
distutils, which is used to compile individual plugins for reForis like
Storage, HaaS and so on.

We no longer need this as we specified host package directly in each
plugin for reForis in the Makefile (present in Turris OS packages
repository) and when we used variable HOST_PYTHON3_PACKAGE_BUILD_DEPENDS
then it is enough to create a text file in packages feed in folder
lang/python/host-pip-requirements/dependency.txt and specify there
package from PyPI with the version and hash.

See for more details:
https://github.com/openwrt/packages/commit/722a5b8efa19fd54728af462441dce08cd7e545b
---
 web/reforis/reforis-distutils/Makefile | 60 --------------------------
 1 file changed, 60 deletions(-)
 delete mode 100644 web/reforis/reforis-distutils/Makefile

diff --git a/web/reforis/reforis-distutils/Makefile b/web/reforis/reforis-distutils/Makefile
deleted file mode 100644
index 6701f518e..000000000
--- a/web/reforis/reforis-distutils/Makefile
+++ /dev/null
@@ -1,60 +0,0 @@
-#
-# Copyright (C) 2018-2019 CZ.NIC z.s.p.o. (http://www.nic.cz/)
-#
-# This is free software, licensed under the GNU General Public License v3.
-# See /LICENSE for more information.
-#
-
-include $(TOPDIR)/rules.mk
-
-PKG_NAME:=reforis-distutils
-PKG_VERSION:=0.1.0
-PKG_RELEASE:=3
-
-PKG_SOURCE_PROTO:=git
-PKG_SOURCE_URL:=https://gitlab.nic.cz/turris/reforis/reforis-distutils.git
-PKG_MIRROR_HASH:=1bd8b65a9528bbdaac356c618577fde46f6cb62d83cdce3f160e9372413b9c50
-PKG_SOURCE_VERSION:=v$(PKG_VERSION)
-
-PKG_MAINTAINER:=CZ.NIC <packaging@turris.cz>
-PKG_LICENSE:=GPL-3.0-or-later
-PKG_LICENSE_FILES:=LICENSE
-
-HOST_PYTHON3_PACKAGE_BUILD_DEPENDS:= \
-	Babel \
-
-HOST_BUILD_DEPENDS:= \
-	python3/host \
-	node/host \
-
-include $(INCLUDE_DIR)/host-build.mk
-include $(INCLUDE_DIR)/package.mk
-include $(TOPDIR)/feeds/packages/lang/python/python3-package.mk
-
-define Package/reforis-distutils
-  SECTION:=web
-  CATEGORY:=Web
-  SUBMENU:=reForis
-  TITLE:=reForis distutils
-  URL:=https://gitlab.nic.cz/turris/reforis/reforis-distutils
-  DEPENDS:=\
-    +python3-light \
-    +python3-babel
-  VARIANT:=python3
-endef
-
-define Package/reforis-distutils/description
-  Distutils to build reforis and its plugins.
-endef
-
-define Host/Compile
-	$(call Build/Compile/HostPy3Mod,,install --prefix="" --root="$(STAGING_DIR_HOSTPKG)")
-endef
-
-Host/Install:=
-
-$(eval $(call HostBuild))
-
-$(eval $(call Py3Package,reforis-distutils))
-$(eval $(call BuildPackage,reforis-distutils))
-$(eval $(call BuildPackage,reforis-distutils-src))
-- 
2.25.1

